examples/examples_nodejs_test.go

```diff
@@ -196,6 +197,32 @@ func TestSecretsInExplicitProviderNode(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestSSHConnNode(t *testing.T) {
+	token := os.Getenv("DIGITALOCEAN_TOKEN")
+	if token == "" {
+		t.Skipf("Skipping test due to missing DIGITALOCEAN_TOKEN environment variable")
+	}
```

Comment:
This has bit us in the past. Will `token` ever be `""` in CI? If not, lets prevent skipping in CI.

Reply:
I agree. However, this is outside the scope of this PR since other tests do this already, and other providers, too.